2025-04-27 01:06:52,944 INFO    MainThread:1817495 [wandb_setup.py:_flush():76] Configure stats pid to 1817495
2025-04-27 01:06:52,944 INFO    MainThread:1817495 [wandb_setup.py:_flush():76] Loading settings from /gpfs/home/wz1492/.config/wandb/settings
2025-04-27 01:06:52,944 INFO    MainThread:1817495 [wandb_setup.py:_flush():76] Loading settings from /gpfs/scratch/wz1492/DL25SP-Final-Project/wandb/settings
2025-04-27 01:06:52,944 INFO    MainThread:1817495 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2025-04-27 01:06:52,944 INFO    MainThread:1817495 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train_jepa.py', 'program': '/gpfs/scratch/wz1492/DL25SP-Final-Project/train_jepa.py'}
2025-04-27 01:06:52,944 INFO    MainThread:1817495 [wandb_init.py:_log_setup():461] Logging user logs to runs/run_20250427-010651/wandb/run-20250427_010652-ncisgcju/logs/debug.log
2025-04-27 01:06:52,944 INFO    MainThread:1817495 [wandb_init.py:_log_setup():462] Logging internal logs to runs/run_20250427-010651/wandb/run-20250427_010652-ncisgcju/logs/debug-internal.log
2025-04-27 01:06:52,945 INFO    MainThread:1817495 [wandb_init.py:init():495] calling init triggers
2025-04-27 01:06:52,945 INFO    MainThread:1817495 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {'data_dir': '/gpfs/scratch/wz1492/data/train', 'epochs': 50, 'batch_size': 32, 'num_hist': 16, 'num_pred': 1, 'lr': 0.0001, 'action_emb_dim': 4, 'encoder_name': 'dinov2_vits14', 'feature_key': 'x_norm_patchtokens', 'predictor_depth': 4, 'predictor_heads': 4, 'predictor_dim_head': 32, 'predictor_mlp_dim': 512, 'predictor_dropout': 0, 'predictor_emb_dropout': 0, 'predictor_pool': 'attn', 'save_path': 'jepa_model_small_sam_0_mse.pth', 'num_workers': 4, 'no_teacher_forcing': False, 'vicreg': False, 'sched_sample_prob': 0.0, 'output_dir': 'runs'}
2025-04-27 01:06:52,945 INFO    MainThread:1817495 [wandb_init.py:init():548] starting backend
2025-04-27 01:06:52,945 INFO    MainThread:1817495 [wandb_init.py:init():552] setting up manager
2025-04-27 01:06:52,948 INFO    MainThread:1817495 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-27 01:06:52,954 INFO    MainThread:1817495 [wandb_init.py:init():558] backend started and connected
2025-04-27 01:06:52,962 INFO    MainThread:1817495 [wandb_init.py:init():636] updated telemetry
2025-04-27 01:06:53,014 INFO    MainThread:1817495 [wandb_init.py:init():667] communicating run to backend with 30 second timeout
2025-04-27 01:06:54,993 WARNING MsgRouterThr:1817495 [router.py:message_loop():76] message_loop has been closed
2025-04-27 01:07:23,016 INFO    MainThread:1817495 [interface_shared.py:_communicate_run():331] couldn't get run from backend
2025-04-27 01:07:23,016 ERROR   MainThread:1817495 [wandb_init.py:init():676] backend process timed out
2025-04-27 01:07:23,017 ERROR   MainThread:1817495 [wandb_init.py:init():685] encountered error: Error communicating with wandb process
For more info see: https://docs.wandb.ai/library/init#init-start-error
